### PR TITLE
Feature/74 reverse colors

### PR DESCRIPTION
**Pull request by StefGehring**
fix trend order of area plot both global and country tabs, added utils function (#74)

### CODE REVIEW

**Comment by StefGehring** [comment]
@riccardoporreca probably an issue with the gitHub action while pushing - could you confirm?

File: R/utils.R [hunk 1]
```diff
@@ -135,6 +135,13 @@ sort_type_by_max <- function(data) {
   c("active", "recovered", "deaths") %>% .[order(sapply(data[.], max))]
 }
 
+#' Sort type harcoded
+#'
+#' @param data data to be sorted
```
Comment (fvitalini) [inline R/utils.R:140]: The documentation is out of date, the new function has no param

**Comment by riccardoporreca** [comment]
See the actual issue https://github.com/miraisolutions/Covid19/issues/74#issuecomment-621764565, where I am sort of challenging this proposed change and wondering whether it was fully tough-through in terms of reduced readability of the log plots.

**Comment by riccardoporreca** [comment]
> @riccardoporreca probably an issue with the gitHub action while pushing - could you confirm?

There was an issue with the cache being originally created from an R 4.x installation. I have forced it to re-install all packages, will revert once done.

**Comment by GuidoMaggio** [comment]
Looks fine to me. I am seeing a warning though.

_Scale for 'colour' is already present. Adding another scale for 'colour',
which will replace the existing scale.
Scale for 'fill' is already present. Adding another scale for 'fill',
which will replace the existing scale._

The log version of course does not look good. I fear it may be more complex to remove it because the log button is used for all the lineplots. 
So it seems to me from a first look on the code. I am fine to keep the log option if this is the case

**Pull request merged**